### ISSUE TITLE
Changes to cart do not appear unless page is refresed (or browser cache is disabled)

### ISSUE THREAD

**Issue by augustin** [pretix/pretix#2019]
**Describe the bug**
On a relatively standard install (MySQL, Redis, but no Memcached), when adding or removing items from a cart, the changes are not visible in the cart unless the page is refreshed, or the changes are made with the browser's cache disabled.

**Expected behavior**
The cart is automatically refreshed when changes are made to it.

**Additional context**
This is in Firefox on Windows, with the ticket shop in test mode.

I can clearly see `cache-control | max-age=600` in the "response headers" and "(cached)" in the request to get the shop/cart page.

**Comment by raphaelm** [comment]
I have never seen or heard of this before. I also have a hard time believing we set a cache control header of that long for *any* request. Can you absolutely exclude any proxy in your setup (nginx config locally or something like cloudflare etc) adds that header? Can you post a video of the exact replication steps? Can you maybe even share the URL to a test setup?

**Comment by augustin** [comment]
Hmm, you are right; I thought I saw the cache-control directly in the gunicorn response headers, but I guess it isn't there; or I was looking at some other page that does have it instead.

The requests are just being proxied with mod_proxy via Apache. I guess something somewhere along the way must be adding that header. Whatever it is does not appear to be very intelligent, as I injected a "cache-control" header via `.htaccess` and there are now two...

So I guess this is an issue on my end somewhere. Sorry for the noise.
